go/vt/vtgate/planbuilder/testdata/from_cases.json

```diff
@@ -3398,32 +3407,40 @@
             "Table": "unsharded_a"
           },
           {
-            "OperatorType": "Join",
-            "Variant": "Join",
-            "JoinColumnIndexes": "L:0,L:1",
-            "TableName": "`user`_user_extra",
+            "OperatorType": "SimpleProjection",
+            "Columns": [
+              1
+            ],
             "Inputs": [
               {
-                "OperatorType": "Route",
-                "Variant": "Scatter",
-                "Keyspace": {
-                  "Name": "user",
-                  "Sharded": true
-                },
-                "FieldQuery": "select `user`.id, `user`.col1 from `user` where 1 != 1",
-                "Query": "select `user`.id, `user`.col1 from `user`",
-                "Table": "`user`"
-              },
-              {
-                "OperatorType": "Route",
-                "Variant": "Scatter",
-                "Keyspace": {
-                  "Name": "user",
-                  "Sharded": true
-                },
-                "FieldQuery": "select 1 from user_extra where 1 != 1",
-                "Query": "select 1 from user_extra",
-                "Table": "user_extra"
+                "OperatorType": "Join",
+                "Variant": "Join",
+                "JoinColumnIndexes": "L:0,L:1",
+                "TableName": "`user`_user_extra",
+                "Inputs": [
```

Comment:
why do we have a simple projection on top of join? Can join handle the projection here?

Reply:
https://github.com/vitessio/vitess/pull/12954#issuecomment-1520004048